ui/app/AppLayouts/Communities/panels/AirdropsSettingsPanel.qml

```diff
@@ -43,11 +43,92 @@ StackView {
     signal registerAirdropFeeSubscriber(var feeSubscriber)
     signal enableNetwork(int chainId)
 
+    Loader {
+        id: assetsModelLoader
+        active: d.loadModels && root.communityTokens
+
+        sourceComponent: SortFilterProxyModel {
+
+            sourceModel: root.communityTokens
+            filters: ValueFilter {
+                roleName: "tokenType"
+                value: Constants.TokenType.ERC20
+            }
+            proxyRoles: [
+                ExpressionRole {
```

Comment:
You could speed those proxy rules even further, by using `FastExpressionRole` with the `expectedRoles` property